src/Microsoft.PowerApps.TestEngine/TestEngine.cs

```diff
@@ -123,6 +123,19 @@ public async Task<string> RunTestAsync(FileInfo testConfigFile, string environme
             {
                 if (TestLoggerProvider.TestLoggers.ContainsKey(testRunId))
                 {
+                    if (!string.IsNullOrEmpty(testRunDirectory))
+                    {
+                        if (!Directory.Exists(testRunDirectory))
+                        {
+                            _fileSystem.CreateDirectory(testRunDirectory);
+                        }
```

Comment:
You would still need to set the outputdirectory this one to properly capture the logs within that 
<img width="500" alt="image" src="https://github.com/microsoft/PowerApps-TestEngine/assets/98551644/16f948d9-3b98-4e9b-b98b-6309b4d399c8">